src/styles/blocks/buy-button.scss

```diff
@@ -0,0 +1,22 @@
+.buy-button {
+  width: 166px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+
+  color: #fff;
+  background-color: $blue-accent;
+  border-radius: 5px;
+
+  &:hover {
+    color: var(--blue-accent);
```

Comment:
Change css var to scss var because its doesn't work on page

Reply:
Thanks! I didn't notice that